### DISCUSSION TITLE
go-runner

### DISCUSSION THREAD

**Discussion by heisetiankong8655** [apache/apisix#6224] [Q&A]
请问用go-runner开发自定义插件，如何获取请求体并编辑内容呢？
我打印了filter中的pkgHTTP.Request并没有看到对应的请求体信息
![image](https://user-images.githubusercontent.com/54174271/151488945-4ae43061-89e3-47be-9393-e67d15842df9.png)

**Comment by spacewander** [comment]
See https://github.com/apache/apisix-go-plugin-runner/issues/52
PR is welcome!
